Gemfile

```diff
@@ -19,4 +19,12 @@ gem 'puppet-lint-unquoted_string-check', :require => false
 if RUBY_VERSION < '2.0'
       gem 'mime-types', '<3.0', :require => false
 end
+
+group :system_tests do
+  if beaker_version = ENV['BEAKER_VERSION']
+    gem 'beaker', *location_for(beaker_version)
+  end
+  gem 'beaker-puppet_install_helper', :require => false
+  gem 'master_manipulator', '1.1.2',  :require => false
```

Comment:
Master Manipulator was fixed so it is no longer necessary to pin to a specific version. I would suggest changing this to ```'master_manipulator', '~> 1.2'```